examples/example-filter-pipeline.json

```diff
@@ -0,0 +1,199 @@
+{
+    "artifact": {
+        "name": "cdap-data-pipeline",
+        "version": "6.1.0-SNAPSHOT",
```

Comment:
should be 6.1.1 since thats the version that has support for this plugin

Reply:
done